<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Outsecure</class>
 <widget class="QWidget" name="Outsecure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>341</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 220);</string>
  </property>
  <widget class="QLabel" name="l_title">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(242, 247, 247);
background: transparent;
font: 25pt &quot;.SF NS Text&quot;;</string>
   </property>
   <property name="text">
    <string>Login Page</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Submit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(160,0,0);
border-style:outset;
border-radcolor: rgb(0, 0, 0);
background-color:rgb(175,238,238);
border-style:outset;
border-radius:10px;
font: 14pt &quot;Arial&quot;;ius:10px;
font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_newuser">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(160,0,0);
border-style:outset;
border-radcolor: rgb(0, 0, 0);
background-color:rgb(175,238,238);
border-style:outset;
border-radius:10px;
font: 14pt &quot;Arial&quot;;ius:10px;
font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>New User</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_username">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>275</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(207, 211, 211);
border-style:outset;
border-radius:10px;
font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="placeholderText">
    <string>Enter UserName</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_password">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(207, 211, 211);
border-style:outset;
border-radius:10px;
font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="placeholderText">
    <string>Enter Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>351</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/bestm/Desktop/IMG/Pokemon.jpg</pixmap>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>l_title</zorder>
  <zorder>btn_Submit</zorder>
  <zorder>btn_newuser</zorder>
  <zorder>txt_username</zorder>
  <zorder>txt_password</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
